docs/tokencache.md

```diff
@@ -2,16 +2,350 @@
 
 ## Introduction
 
-> PENDING
+The token cache implementation for MSAL, ADAL and Android broker is found in the Android Common Library.  
```

Comment:
If you have time, I would suggest adding a diagram to show the relationship between classes in this doc (and how the flow works). I think that'll greatly help newcomers.

Reply:
Done.